HUGODEHSIGN/my-otaku-opinions
# 57 create login page and signup page

## Purpose

Please provide a detailed explanation of the PR. What existing problem does it solve?

Fixes #57 

## Changes

Please provide an overview of code changes, especially key files and areas of significant change.

- path/to/file
  - controllers/index.js
  - controllers/login-routes.js
  - controllers/signup-routes.js
  - server.js
  - views/layouts/main.handlebars
  - views/login.handlebars
  - views/signup.handlebars

## Steps to Reproduce or Test

Demonstrate that the pull request is functional. For example: the exact commands to be run and their output; screenshots if the PR modifies the UI.

- clone the branch
- start server
- go to (/signup) and (/login) routes and check that they work
